source/interprocedural_analyses/taint/modelParser.ml

```diff
@@ -1943,7 +1925,13 @@ let add_taint_annotation_to_model
     annotation
   =
   let open Core.Result in
-  let annotation_error = invalid_model_error ~path ~location ~name:model_name in
+  let annotation_error = model_verification_error ~path ~location in
+  let annotation_error_map error_message =
```

Comment:
nitpick: Give those function better names.
`annotation_error` could be `model_verification_error`. It is a shortcut.
`annotation_error_map` could be `invalid_model_for_taint`.